.github/workflows/pull-request.yml

```
@@ -253,6 +253,45 @@ jobs:
+        run: |
+          timeout=60
+          while ! nc -z localhost 8080 && [ $timeout -gt 0 ]; do
+            sleep 1
+            timeout=$((timeout-1))
+          done
+          if [ $timeout -eq 0 ]; then
+            echo "Timeout waiting for application to start"
+            exit 1
+          fi
+          curl --fail --silent http://localhost:8080/index.html || exit 1
+    
```

yamllint

[error] trailing spaces

(trailing-spaces)